frontend/app/routes/login.tsx

```diff
@@ -13,10 +20,13 @@ export const action: ActionFunction = async ({ request }) => {
       username: formData.get("email") as string,
       password: formData.get("password") as string,
     });
+    const cookies = [getCookieToDelete("isFirstTime"), `token=${resp.access_token}`];
+    const headers = cookies.reduce((headers, curr) => {
+      headers.append("Set-Cookie", curr);
+      return headers;
+    }, new Headers());
```

Comment:
Make this into an util, maybe something like this:
```typescript
export function makeCookies(...cookies: string)
```

so we can use it like this:
```typescript
const cookieHeader = makeCookies(cookie('foo', 'bar'), deleteCookie('korwin'))
```